apps/utils/db-query-checker/src/lib.rs

```diff
@@ -347,18 +349,18 @@ async fn select_bene_ids_by_part_d_contract_id_and_year_month(
     // Create the query.
     let bene_ids_query = sqlx::query(
         DATABASE_QUERY_SQL
-            .get(&DatabaseQuery::SelectBeneIdsByPartDContractIdAndYearMonth)
+            .get(&DatabaseQuery::BeneIdsByPartDContractIdAndYearMonth)
             .unwrap(),
     )
-    .bind(&year_month)
-    .bind(&partd_contract_id)
-    .bind(&BENES_PAGE_SIZE);
+    .bind(year_month)
+    .bind(partd_contract_id)
+    .bind(BENES_PAGE_SIZE);
```

Comment:
Also not sure what this change is about.

Reply:
This changes (and changes like them) resulted from suggestions made by [clippy](https://doc.rust-lang.org/stable/clippy/index.html), specifically erroring on a [Needless Borrow](https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow).